clevertap-core/src/main/java/com/clevertap/android/sdk/inapp/evaluation/EvaluationManager.kt

```diff
@@ -51,13 +67,13 @@ internal class EvaluationManager(
     private val templatesManager: TemplatesManager
 ) : NetworkHeadersListener {
 
-    // Internal list to track server-side evaluated campaign IDs.
+    // Internal map to track server-side evaluated campaign IDs. This map is used to identify the evaluatedIDs for raised and profile events individually
     @VisibleForTesting
-    internal var evaluatedServerSideCampaignIds: MutableList<Long> = ArrayList()
+    internal var evaluatedServerSideCampaignIds: MutableMap<String, MutableList<Long>> = mutableMapOf(Constants.RAISED to mutableListOf(), Constants.PROFILE to mutableListOf())
```

Comment:
Since we changing this from list to map is migration required as we use Shared prefs to load this.

Reply:
MIgration handled in `InAppStore.kt`
@CTLalit